css/style.css

```diff
@@ -1,9 +1,21 @@
+/*
+ * Reset
+ */
+
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
 body {
     background-color: #2e2e2e;
     color: #CCC;
     font-family: 'Droid Sans', "Lucida Grande", "Lucida Sans", "Liberation Sans", sans-serif;
     font-size: 0.8em;
```

Comment:
I don’t know extactly where to place this comment, but with this PR, the font is noticeably smaller (12px instead of 15px) than before. Can we keep the current font size?

Reply:
The font is smaller because the base font-size of the body element on this branch is still set to 0.8em. The #content rule has a 1.2em font-size rule which ended up normalizing the font size to 1em, but with the removal of #content on the main pages you end up seeing 0.8em for now. Once the typography branch is merged, the base font-size will be 1em and things should look like they used to. #content will need some adjusting afterwards, like setting the font-size back to 1em and removing the redundant line-height, but I felt like that wasn't an appropriate change for this branch,